lib/models/depot.dart

```diff
@@ -63,6 +82,19 @@ class Depot {
     );
   }
 
+  // Define a function that insert the array depots into database with transaction
+  static Future<void> txInsertLot(dynamic database, List<Depot> depots) async {
+    final db = await database;
+    await db.transaction((txn) async {
+      final batch = txn.batch();
+      for (var depot in depots) {
+        batch.insert('depot', depot.toMap(),
+            conflictAlgorithm: ConflictAlgorithm.ignore);
+      }
+      await batch.commit(noResult: true);
+    });
+  }
```

Comment:
Why do we need this function? does it have any effect on performance?

Reply:
Because there was a performance issue when inserting multiple data at once